xml/Microsoft.Extensions.Caching.Distributed/DistributedCacheExtensions.xml

```diff
@@ -89,7 +89,7 @@
         <param name="key">The key to get the stored data for.</param>
         <summary>Asynchronously gets a string from the specified cache with the specified key.</summary>
         <returns>A task that gets the string value from the stored cache key.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>This method stores all non-usage exceptions that its synchronous counterpart can throw in the task it returns. If an exception is stored during execution, it will be thrown when you await that task. Usage exceptions, such as <see cref="T:System.ArgumentException" />, are still thrown synchronously. For the stored exceptions, see the exceptions thrown by <see cref="M:Microsoft.Extensions.Caching.Distributed.DistributedCacheExtensions.GetString(Microsoft.Extensions.Caching.Distributed.IDistributedCache,System.String)" />.</remarks>
```

Comment:
"If an exception is stored during execution, it will be thrown when you await that task." =>
"If an exception is stored into the returned task, that exception will be thrown when the task is awaited."